transaction/core/src/tx_summary.rs

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! The TxSummary is meant to reduce the complexity of implementing a hardware
+//! wallet. (See MCIP 52 also.)
```

Comment:
Nit: "also" seems premature here
```suggestion
//! wallet. (See MCIP 52)
```